### PR TITLE
EthersFi implementation

### PR DESCRIPTION
**Pull request by cosminobol**
### Summary
Ethersfi protocol split development

### CODE REVIEW

**Review by OisinKyne — COMMENTED** [review]
Oops I left these pending my bad, can we rename "EthersFi" to "EtherFi" everywhere?

File: src/ethersfi/ObolEthersfiSplitFactory.sol [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import {LibClone} from "solady/utils/LibClone.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import "./ObolEthersfiSplit.sol";
+
+/// @title ObolEthersfiSplitFactory
+/// @author Obol
+/// @notice A factory contract for cheaply deploying ObolEthersfiSplit.
+/// @dev The address returned should be used to as reward address for EthersFi
+contract ObolEthersfiSplitFactory {
```
[the remark this answers is not in the document]
Comment (cosminobol) [reply src/ethersfi/ObolEthersfiSplitFactory.sol:12]: @OisinKyne  yes sure; I will do that today and push

**Comment by cosminobol** [comment]
> Oops I left these pending my bad, can we rename "EthersFi" to "EtherFi" everywhere?

No worries! Renamed everything to `EtherFi`

**Comment by cosminobol** [comment]
Included in https://github.com/ObolNetwork/obol-splits/pull/116

**Pull request closed without merging**